devito/finite_differences/differentiable.py

```diff
@@ -402,6 +402,11 @@ class Mod(DifferentiableOp, sympy.Mod):
     __new__ = DifferentiableOp.__new__
 
 
+class EvalDiffDerivative(DifferentiableOp, sympy.Add):
+    __sympy_class__ = EvalDerivative
```

Comment:
why not just `(DifferentiableOp, EvalDerivative):`

Reply:
Tried, it works, so switching to your suggestion